### DISCUSSION TITLE
What does this mean?

### DISCUSSION THREAD

**Discussion by Reyororo** [KadeArchive/Kade-Engine#1990] [Q&A] [answered]
kade engine 1.7 when I run lime test windows with kade engine 1.7, I get this text

【Could not find haxelib "hxvm-luajit", does it need to be installed?】

**Comment by EliteMasterEric** [comment, marked as answer]
The engine has a new dependency, see the build guide (which has been updated):
https://github.com/KadeDev/Kade-Engine/blob/stable/docs/building.md
